### ISSUE TITLE
Qt 5.12: test_follow_a_manually_selected_link fails

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#4250]
With the current Qt 5.12 tip (69-based):

```
FAIL tests/end2end/features/test_search_bdd.py::test_follow_a_manually_selected_link
FAIL tests/end2end/features/test_search_bdd.py::test_follow_a_manually_selected_link_in_a_new_tab
```

I think this is the same situation we had with Qt 5.10 before #3624: Selecting via a search or manually works, but selecting via JS (`search_select.js`) and then `:follow-selected` doesn't seem to work.

Also see #2635.

@jgkamat want to take a look at that? If you don't want to build Qt from source, you can get the Qt 5.12 Alpha via the binary installer, but that's still 67-based. The beta 1 (which will be 69-based) is planned for October 2nd.

**Comment by jgkamat** [comment]
[here's the problem](https://github.com/chromium/chromium/commit/1c0152d0297a8341498a04087587b26ee59fbdda#diff-2e7df0a2c8430a62e64d499c0bba61a1R170), baseNode is now anchorNode. So, I'm guessing we need to set the selection differently in tests now....

**Comment by The-Compiler** [comment]
Good catch! FWIW as long as this doesn't affect caret mode (and I think currently it doesn't, as we clear the selection when leaving it), I'd also be fine with skipping those tests.

**Comment by jgkamat** [comment]
I think I can modify the tests a bit to select text more like search/humans do (by starting at the bottom most text, not just the link itself). I'll submit a patch for that soon :)
